source/javascripts/all.js

```diff
@@ -1 +1,7 @@
 //= require_tree .
+$(function() {
+    $(document).on("click",".tool", function(){
```

Comment:
I'll manually move this into the tooling logic afterwards, since there is a similar click handler that can be tweaked to capture whole row.